.github/CODEOWNERS

```diff
@@ -1,20 +1,22 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 # Owners bear a responsibility to the organization and the users of this
-# application. Repository administrators have the ability to merge pull 
-# requests that have not yet received the requisite reviews as outlined 
-# in this file. Do not force merge any PR without confidence that it 
-# follows all policies or without full understanding of the impact of 
+# application. Repository administrators have the ability to merge pull
+# requests that have not yet received the requisite reviews as outlined
+# in this file. Do not force merge any PR without confidence that it
+# follows all policies or without full understanding of the impact of
 # those changes on build, release and publishing outcomes.
 
 *                                    @MetaMask/extension-devs
+**/snaps/**                          @MetaMask/snaps-devs
+**/flask/**                          @MetaMask/snaps-devs
 development/                         @MetaMask/extension-devs @kumavis
-lavamoat/                            @MetaMask/supply-chain
```

Comment:
The base branch is what gets used for CODEOWNER requirements, so we need a supply chain approval here (or we need to override this).

Worth noting for next time that we will need snaps team approvals for any LavaMoat policy changes going forward due to the `**/flask/**` entry. That doesn't seem ideal either.